cmd/scheduled-feed/main.go

```diff
@@ -22,10 +22,17 @@ type FeedHandler struct {
 	scheduler *scheduler.Scheduler
 	pub       publisher.Publisher
 	delta     time.Duration
+	lastPoll  time.Time
 }
 
 func (handler *FeedHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
```

Comment:
This function is getting quite loaded with logic, we should break the cutoff calculation out to start to simplify this.

Reply:
Sure I don't disagree there, I've broke this out into a value receiver func